### DISCUSSION TITLE
Automating the opening of a new decision

### DISCUSSION THREAD

**Discussion by Monstarrrr** [Monstarrrr/rebutify#67] [Ideas & Suggestions]
Currently the process look like this:  
![image](https://github.com/Monstarrrr/rebutify/assets/31903668/73570f63-4740-40ef-99cd-736482243b5f)

The problem is if we want to centralize our workspace to include non-technical tasks as part of our github issues & github project board, we need to simplify the creation of new decision files for non-tech members as much as possible.  

@seporterfield Could you report on how feasible it would be to automate and simplify this process ?

**Comment by seporterfield** [comment]
One possible partially automated workflow:
- contributor creates issue/
discussion with decision in body
- github action triggers automatically 
- a branch with the new decision doc has been automatically created and turned into PR draft
